packages/fast-check/postbuild/main.mjs

```diff
@@ -67,10 +67,40 @@ fs.readFile(path.join(__dirname, '../package.json'), (err, data) => {
     from: [/__PACKAGE_VERSION__/g, /__COMMIT_HASH__/g],
     to: [packageVersion, commitHash],
   });
-  if (dTsReplacement2.length === 1 && dTsReplacement[0].hasChanged) {
+  if (dTsReplacement2.length === 1 && dTsReplacement2[0].hasChanged) {
     console.info(`Package details added onto d.ts version for esm`);
   }
 
+  function reportArrayReplace(results) {
+    for (const result of results.filter((x) => x.hasChanged)) {
+      if (result.numReplacements === 1) {
+        console.info(`Stripped ${result.numReplacements} generic from typed array for `, result.file);
+      } else {
+        throw new Error(
+          `We expected to only replace 1 generic for ${result.file}, but instead replaced ` + result.numReplacements,
```

Comment:
Since we're doing a blind regex replace, I made it intentionally fragile so it'll throw if anything weird/unexpected happens.